src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2263,12 +2275,14 @@ private void doEncodeConnack(
                 propertiesSize = mqttProperty.limit();
             }
 
+            int flags = 0x01;
+
             final int propertiesSize0 = propertiesSize;
             final MqttConnackFW connack =
                     mqttConnackRW.wrap(writeBuffer, FIELD_OFFSET_PAYLOAD, writeBuffer.capacity())
                                        .typeAndFlags(0x20)
                                        .remainingLength(3 + propertiesSize0)
-                                       .flags(0x00)
+                                       .flags(flags)
```

Comment:
Let's use a constant like `CONNACK_SESSION_PRESENT` instead of `flags` for improved readability.